lib/npm/node-platform.ts

```diff
@@ -24,6 +24,7 @@ export const knownUnixlikePackages: Record<string, string> = {
   'linux ia32 LE': 'esbuild-linux-32',
   'linux mips64el LE': 'esbuild-linux-mips64le',
   'linux ppc64 LE': 'esbuild-linux-ppc64le',
+  'linux s390x BE': 'esbuild-linux-s390x',
```

Comment:
I just realized that this is our first big-endian platform! I think everything should work fine but we'll see. If something doesn't work about the published package or install script, we can fix it later.